styles.css

```diff
@@ -559,6 +598,161 @@ p {
   position: absolute;
 }
 
+/* popup details */
+
+#popup-window {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  overflow: auto;
+  background-color: white;
+}
+
+#overlay {
+  width: 375px;
+  height: 882px;
+  left: 0;
+  top: 0;
+  background: #c1c7d0;
+  mix-blend-mode: multiply;
+}
+
+#card-works-container {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  gap: 12px;
+  width: 343px;
+  height: 836px;
+  left: calc(50% - 343px / 2);
+  top: 20px;
+  background: white;
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
- [ ] Also note that your project is not responsive on both mobile and desktop design so make sure you adjust your style to remove every horizontal scroll.

| Your Design(On mobile) |Figma Design (On mobile) |
| :---:   | :---: | 
| ![Screenshot from 2023-01-07 00-56-05](https://user-images.githubusercontent.com/92303891/211123913-fb423db3-6264-4d03-84b2-9cbb5739a64e.png) | ![Screenshot from 2023-01-07 01-03-15](https://user-images.githubusercontent.com/92303891/211123995-c1fa41ff-2671-44a7-a255-526d1cae0982.png)|

| Your Design(On Desktop) |Figma Design (On Desktop) |
| :---:   | :---: | 
| ![Screenshot from 2023-01-07 01-05-43](https://user-images.githubusercontent.com/92303891/211124116-c39ca5fb-8fc1-4cfb-b1c8-787753714bad.png) ![Screenshot from 2023-01-07 01-06-11](https://user-images.githubusercontent.com/92303891/211124131-3599807a-0c0f-480d-8736-3868912f1d27.png)|![Screenshot from 2023-01-07 01-04-58](https://user-images.githubusercontent.com/92303891/211124141-93729e2d-8328-4dba-8cb3-fb48b74465fe.png) |